src/state/user/hooks.tsx

```diff
@@ -122,14 +125,23 @@ export function useIsAcceptedTerm(): [boolean, (isAcceptedTerm: boolean) => void
 
 export function useDegenModeManager(): [boolean, () => void] {
```

Comment:
Why not separate this hook into two separate hooks? This way, if we need to use these hooks on the same page in the future, it will be more explicit and easier to manage.